symbionic/gui/kivy_gui.py

```diff
@@ -148,6 +156,29 @@ def save(self, path, filename):
         self.dismiss_popup()
 
 
+class clsCheckBox:
+    def __init__(self, Logic, cbox_id, number_of_cbs, CB_group):
+        self.Logic = Logic
+        self.cbox_id = cbox_id
+        self.number_of_cbs = number_of_cbs
+        self.CB_group = CB_group
+        self.cbs = []
+        self.ylim = (-150,150)
+
+    def init_cbs(self):
+        for g in range(self.number_of_cbs):
+            self.add_cb(self.cbox_id)
+
+    def add_cb(self, box_id):
```

Comment:
Same here; changed cb to check_box
```suggestion
    def add_check_box(self, box_id):
```

Reply:
Ok, just fix this everywhere.. I see lots of cb, cbs, cbox, CB_group, etc

Reply:
Done!